### ISSUE TITLE
API cuts off content data 

### ISSUE THREAD

**Issue by alexcarcoana** [bzarras/newsapi#38]
Hi,

I noticed the API always cuts off content data, leaving something like this at the end of the string [+2219 chars]

Is there any way to tell the API to give the full content and not shorten it?

**Comment by mashegoindustries** [comment]
Hi,

According to newsapi.org documentation
"The unformatted content of the article, where available. This is truncated to 200 chars."

In simple terms, it means they dont want you to be scrapping people's websites without permission, you should always show link to full article

**Comment by YannyTwoo** [comment]
Yep, as @mashegoindustries said. The whole article can't be displayed. It's a NewsAPI feature itself

**Comment by Boscop** [comment]
Is there no way to get the full article content via the API?
